app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    return (
+        f"Friends should buy {masks_to_buy} masks" if masks_to_buy
+        else f"Friends can go to {cafe.name}"
+    )
```

Comment:
Avoiding usage of ternary operator will make it more readable. It will also take one line less